### PR TITLE
Adding dashboard page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Pages/DashboardPage/DashboardPage.jsx

## What not to check
- [ ] Do Not Check Any Other Files

Live Preview,
https://stormkeep.netlify.app/

### CODE REVIEW

File: src/Pages/DashboardPage/DashboardPage.jsx [hunk 1]
```diff
@@ -1,33 +1,69 @@
...
+                  </div>
+                </>
+              ) : null}
+              {route === "settings" ? (
+                <>
+                  <h2>Account Settings</h2>
+                  <Link to="/" element={<LandingPage />}>
+                    {encodedToken && (
+                      <button
+                        onClick={() => signout()}
```
Comment (foolhardy21) [inline src/Pages/DashboardPage/DashboardPage.jsx:58]: suggestion - as `signout` does not accept any arguments, you can just write `onClick = {signout}`

**Review by foolhardy21 — APPROVED** [review]
code looks good.

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -1,45 +1,70 @@
...
 import { useAuthContext } from "../../Context/Auth.context";
+import { useReducerContext } from "../../Context/Reducer.context";
+import { useThemeContext } from "../../Context/Theme.context";
 import { LoginBox } from "../../Pages/AuthPage/ProfileComponents";
 import "./Header.css";
 
 export const Header = () => {
+  const { theme, toggleLightDarkTheme } = useThemeContext();
   const user = localStorage.getItem("StormKeepUser");
   const encodedToken = localStorage.getItem("StormKeepToken");
```
Comment (ShekharDhangar) [inline src/Components/Header/Header.jsx:11]: could make a custom hook for  using localstorage.

**Review by ShekharDhangar — APPROVED** [review]
Great work! Apart from suggested change, rest looks good.

File: src/Components/Header/Header.jsx [hunk 3]
```diff
@@ -1,45 +1,70 @@
 import { Link, useLocation } from "react-router-dom";
 import { useAuthContext } from "../../Context/Auth.context";
+import { useReducerContext } from "../../Context/Reducer.context";
+import { useThemeContext } from "../../Context/Theme.context";
 import { LoginBox } from "../../Pages/AuthPage/ProfileComponents";
 import "./Header.css";
 
 export const Header = () => {
+  const { theme, toggleLightDarkTheme } = useThemeContext();
```
Comment (ShekharDhangar) [inline src/Components/Header/Header.jsx:9]: Appreciate that you implemented dark theme

**Review by ShekharDhangar — APPROVED** [review]

File: src/Pages/DashboardPage/DashboardPage.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/DashboardPage/DashboardPage.jsx:58]: yes thanks for pointing it out. I actually forgot to change it back!

File: src/Components/Header/Header.jsx (hunk 3 above)
Comment (b0llu) [reply src/Components/Header/Header.jsx:9]: Thanks!

File: src/Components/Header/Header.jsx (hunk 2 above)
Comment (b0llu) [reply src/Components/Header/Header.jsx:11]: Will think on this

**Pull request merged**